sql/migrations/002_tags.sql

```diff
@@ -0,0 +1,13 @@
+-- +migrate Up
+
+ALTER TABLE "task" ADD COLUMN "tags" JSONB;
+ALTER TABLE "task_template" ADD COLUMN "tags" JSONB;
+
+CREATE INDEX "task_tags_idx" ON "task" USING gin (tags jsonb_path_ops);
```

Comment:
A comment seems worthwhile here to explain the index type choice, e.g. link to doc or similar?

Reply:
Ok.